core/src/main/resources/error/error-classes.json

```diff
@@ -3684,7 +3684,7 @@
       "<message>. If necessary set <ansiConfig> to false to bypass this error."
     ]
   },
-  "_LEGACY_ERROR_TEMP_2003" : {
+  "CANNOT_ZIP_MAPS" : {
```

Comment:
Highly likely, you need to reorder the error class since the error classes in the file should be ordered alphabetically.